File: c++/control_flow/if_statement/if_statement.cpp
#include <iostream>

int main()
{
  if (1 > 2)
    std::cout << "1 > 2" << std::endl;

  if (2 > 1)
    std::cout << "2 > 1" << std::endl;

  std::cout << std::endl;

  bool condition;

  condition = false;

  if (condition)
    std::cout << "False: ok" << std::endl;

  condition = true;

  if (condition)
    std::cout << "True: ok" << std::endl;

  std::cout << std::endl;
}

File: c#/control_flow/if_statement/IfStatement.cs
class IfStatement {
  static void Main(string[] args) {
    if (1 > 2)
      System.Console.WriteLine("1 > 2");

    if (2 > 1)
      System.Console.WriteLine("2 > 1");

    System.Console.WriteLine();

    bool condition;

    condition = false;

    if (condition)
      System.Console.WriteLine("false: ok");

    condition = true;

    if (condition)
      System.Console.WriteLine("true: ok");

    System.Console.WriteLine();
  }
}
